app/middleware.go

```diff
@@ -74,8 +76,14 @@ func (l LoggerMiddleware) Serve(next echo.HandlerFunc) echo.HandlerFunc {
 
 		startTime = time.Now()
 
+		metricTagMap := make(map[string]interface{})
+		c.Set("metricTagsMap", metricTagMap)
```

Comment:
It would be nice to match the key with the variable name. Replace `metricTagMap` with `metricTagsMap`.

Reply:
Done, mrs bitbanshee